src/Runner.Worker/JobExtension.cs

```diff
@@ -131,7 +131,7 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                     // Temporary hack for GHES alpha
                     var configurationStore = HostContext.GetService<IConfigurationStore>();
                     var runnerSettings = configurationStore.GetSettings();
-                    if (!runnerSettings.IsHostedServer && !string.IsNullOrEmpty(runnerSettings.GitHubUrl))
+                    if (string.IsNullOrEmpty(context.GetGitHubContext("url")) && !runnerSettings.IsHostedServer && !string.IsNullOrEmpty(runnerSettings.GitHubUrl))
                     {
                         var url = new Uri(runnerSettings.GitHubUrl);
                         var portInfo = url.IsDefaultPort ? string.Empty : $":{url.Port.ToString(CultureInfo.InvariantCulture)}";
```

Comment:
do we need?
```
context.SetGitHubContext("graphql_url", xxx);	                        
```

Reply:
nah this is just alpha back compat. After the real URLs are sent in the payload we can delete this code.